src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -200,30 +249,29 @@ const ChartsHolder = ({ projections, region, chartId }: ChartsHolderProps) => {
           <VulnerabilitiesBlock scores={ccviScores} region={region} />
         </LocationPageBlock>
         <LocationPageBlock>
-          <Recommend
-            introCopy={recommendationsIntro}
-            recommendations={recommendationsMainContent}
-            locationName={region.fullName}
-            shareUrl={recommendsShareUrl}
-            shareQuote={recommendsShareQuote}
-            recommendationsRef={recommendationsRef}
-            feedbackFormUrl={recommendationsFeedbackForm}
-            fedLevel={getFedLevel(projections.primary)}
-            harvardLevel={getHarvardLevel(projections.primary)}
-            harvardModalLocationCopy={recommendationsHarvardModalCopy}
-            fedModalLocationCopy={recommendationsFedModalCopy}
-          />
+          {!projections ? (
+            <LoadingScreen />
+          ) : (
+            <Recommendations
+              projections={projections}
+              recommendationsRef={recommendationsRef}
+            />
+          )}
           {ALL_METRICS.map(metric => (
             <ErrorBoundary>
-              <ChartBlock
-                key={metric}
-                metric={metric}
-                projections={projections}
-                chartRef={metricRefs[metric]}
-                isMobile={isMobile}
-                region={region}
-                stats={stats}
-              />
+              {!projections ? (
```

Comment:
It seemed better to have a separate loading thing for each chart rather than one loading thing for the whole bunch of them but I'm happy to  hange this if preferred.